app/views/admin/dashboard/index.html.erb

```diff
@@ -1 +1,18 @@
 <%= content_tag :h1, "Welcome #{current_user.name}!" %>
+
+<%= content_tag :ul, id: 'all-orders' do %>
+  <%= content_tag :h3, "Orders:" %>
```

Comment:
Great use of the rails syntax here! There are a few things I had written that could use some refactoring to match this style!